spec/rspec/rails/example/view_example_group_spec.rb

```diff
@@ -213,6 +222,7 @@ def controller
     describe "#view" do
       let(:view_spec) do
         Class.new do
+          def _view; end # Stub method
```

Comment:
Whats this for?

Reply:
It's to satisfy [this](https://github.com/rspec/rspec-rails/pull/2264/files/87aee32e014aed4c818156a9c473c5de8682b8cc#diff-0cdf276129b0b83c6314283584c4df6eR232).

Reply:
This link doesn't seem to work